rdflib/plugins/serializers/turtle.py

```diff
@@ -77,8 +77,8 @@ def orderSubjects(self):
             members = list(self.store.subjects(RDF.type, classURI))
             members.sort()
 
+            subjects = members
             for member in members:
-                subjects.append(member)
```

Comment:
this is not equivalent: if there are multiple `classURI` then before `subjects` contained all members, now it will only contain those of the last...

Reply:
line 74 is `subjects = []` so I think this is equivalent

Reply:
and because topClasses has one element

Reply:
assuming that selfClasses is always [RDFS.Class] the outer loop can be removed and subject = members is correct